PRINCIPLES.md

```diff
@@ -0,0 +1,259 @@
+  - [3. Continuous state reconciliation](#3-continuous-state-reconciliation)
+  - [4. Operations through declaration](#4-operations-through-declaration)
+- [See Also](#see-also)
+
+## Summary
+
+GitOps is a set of principles for operating and managing software systems.
+
+When using GitOps, the desired state of a system or subsystem is defined declaratively as versioned, immutable data, and the running system's configuration is continuously derived from this data.
+
+GitOps principles were derived from modern software operations but are rooted in pre-existing and widely adopted best practices. These principles are:
+
+1. [**The principle of declarative configuration**](#1-declarative-configuration)
+
+    A system managed by GitOps must have its _Desired State_ expressed declaritively as data in a format writable and readable by both humans and machines.
+
+2. [**The principle of immutable configuration versions**](#2-immutable-configuration-versions)
+
+    _Desired State_ is stored in a way that supports versioning, immutability of versions, and retains a complete version history.
+    We call systems that store desired state in this way _State Stores_.
+
+3. [**The principle of continuous state reconciliation**](#3-continuous-state-reconciliation)
+
+    Software agents continuously, and automatically, compare a system's _Actual State_ to its _Desired State_.
+    If the actual and desired states differ, automated actions are immediately attempted to reconcile them.
+    These differences could be due to the actual state drifting from the desired state, or the desired state changing intentionally.
+
+4. [**The principle of operations through declaration**](#4-operations-through-declaration)
+
+    The mechanism through which change is applied to a system by either a human operator or another system is through the creation of a new declarative version of the desired state in the state store, not through directly interacting with the running system.
+
+## Introduction
+
+The systems that we manage vary widely; from battery-powered devices driven by microcontrollers to globally distributed systems with millions and sometimes billions of users.
+
+What is common across all systems is the need for human operators to make changes to the running state, it is during change that systems are most at risk of failure.
+
+GitOps is a framework in which all change is applied to a system in a consistent mechanism that both reduces risk and improves the situational awareness of human operators working inside the system.
+
+GitOps can be applied to managing large systems or applied only to the smaller sub-system.
+
+See also the [Rationale for GitOps](RATIONALE.md).
+
+## Scope
+
+GitOps concerns the interaction between humans and technical systems, and between technical systems.
+GitOps is not concerned with processes of human decision making or organisation, only how human decisions about technical systems are recorded and applied.
```

Comment:
I strongly disagree with this line. GitOps is all about human decision-making, i.e. a human deciding to make a change to a system and another human deciding that the change is of low enough risk to be merged. 

Part of the areas needing improvement is how human-decision making can be augmented by a better understanding of *how* a change could affect a system. (Tests / linting play a role here but usually disconnected from the system) i.e Will scaling down cause production workloads to be evicted or scaling up exceed my organizational budget

GitOps also has scaling problems in large organizations, and one of the goals of the WG should be to try and solve them e.g. 
* How do multiple teams in an organization interact on a single repo, 
* How is a system decomposed into multiple repositories 
* How are systematic changes across systems and repo's implemented and monitored
* How are changes promoted through environments while respecting change windows and organizational change policies. 

```suggestion
```

Reply:
If I understand your comment correctly, this is not what was meant here. This is to clarify that the scope of GitOps is to define and reconcile technical systems, not human systems. Humans are always on some level in control of the automated processes. No [HAL](https://en.wikipedia.org/wiki/HAL_9000) or [Skynet](https://en.wikipedia.org/wiki/Skynet_(Terminator)) here LOL 😛  But seriously, we could probably word this better.